data/map systems.txt

```diff
@@ -4993,6 +4993,70 @@ system Alpheratz
 			distance 285
 			period 12.909
 
+system Alpherg
+	pos -1009.44 272.194
+	government Pirate
+	arrival 500
+	habitable 425
+	haze _menu/haze-33
+	link Fumalasamakah
+	asteroids "medium rock" 3 2.97
+	asteroids "small metal" 3 3.195
+	trade Clothing 305
+	trade Electronics 878
+	trade Equipment 442
+	trade Food 221
+	trade "Heavy Metals" 860
+	trade Industrial 900
+	trade "Luxury Goods" 1280
+	trade Medical 869
+	trade Metal 280
+	trade Plastic 286
+	fleet "Small Southern Pirates" 1000
+	fleet "Small CCOR" 1500
+	fleet "Large Southern Pirates" 2000
+	fleet "Large CCOR" 3000
+	object
+		sprite star/k5
+		period 25
+	object "Jakobsen"
+		sprite planet/ocean3-b
+		distance 445.1
+		period 90
+		offset 84
+		object "Mendez Station"
+			sprite planet/station8
+				scale 0.5
+			distance 171.4
+			period 61.454
+	object
+		sprite planet/cloud8-b
+		distance 1817.57
+		period 1503.50
+		offset 55
```

Comment:
At the moment, whether or not Mendez orbits the star is up for debate. Generally we avoid human space stations orbiting a star rather than a planet as a matter of feat, but I'll be bringing it up again next time we discuss it.